### PR TITLE
Add shopify tables. Closes #23

### LINKED ISSUE #12: Add table shopify_product_variant
**References**
https://pkg.go.dev/github.com/bold-commerce/go-shopify/v3#VariantServiceOp.List
https://pkg.go.dev/github.com/bold-commerce/go-shopify/v3#VariantServiceOp.Get
https://pkg.go.dev/github.com/bold-commerce/go-shopify/v3#VariantServiceOp.ListMetafields

### LINKED ISSUE #23: Add new tables to shopify plugin
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Priyanka-Chatterjee-2000**
* Add shopify_product

* Add more columns to table shopify_product and example queries to docs

* Add limit

* doc update

* Update table name

* Add metafields

* Update product.go

* update descriptions

* Update description of product

* fix extra spaces

* Update doc queries

* Update doc query

* Add table shopify_product_variant. Closes #12 (#17)

* Add shopify_product_variant

* Update shopify_product_variant

* Remove shouldRetryError function

* Add shopify product variant description

---------

# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: shopify/table_shopify_product_variant.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+				Transform:   transform.FromField("AdminGraphqlAPIID"),
+			},
+		},
+	}
+}
+
+func listProductVariant(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	conn, err := connect(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("listProductVariant", "connection_error", err)
```
Comment (madhushreeray30) [inline shopify/table_shopify_product_variant.go:175]:
```suggestion
		plugin.Logger(ctx).Error("listProductVariants", "connection_error", err)
```

File: README.md [hunk 2]
```diff
@@ -1 +1,108 @@
...
+
+Use SQL to query products, order, customers and more from Shopify.
+
+- **[Get started →](https://hub.steampipe.io/plugins/turbot/shopify)**
+- Documentation: [Table definitions & examples](https://hub.steampipe.io/plugins/turbot/shopify/tables)
+- Community: [Slack Channel](https://steampipe.io/community/join)
+- Get involved: [Issues](https://github.com/turbot/steampipe-plugin-shopify/issues)
+
+## Quick start
+
```
Comment (madhushreeray30) [inline README.md:13]:
```suggestion

### Install

```

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+| 5367225254183 | #1009 | russel.winfield@example.com | 105 Victoria St | Toronto | Canada  | M5C1N7   |
+| 5367225221415 | #1008 | russel.winfield@example.com | 105 Victoria St | Toronto | Canada  | M5C1N7   |
++---------------+-------+-----------------------------+-----------------+---------+---------+----------+
+```
+
+## Documentation
+
+- **[Table definitions & examples →](/plugins/turbot/shopify/tables)**
+
+## Get started
```
Comment (madhushreeray30) [inline docs/index.md:49]:
```suggestion
## Quick start
```

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  # Can also be set with the SHOPIFY_SHOP_NAME environment variable.
+  # shop_name = "theshop"
+}
+```
+
+- `shop_name` - Shopify shop_name refers to the unique name that a merchant chooses for their online store built on the Shopify platform.
+
+- `token` - Shopify api_token is a secure and unique identifier that is used to authenticate and authorize a third-party app or service to access a Shopify store's data and perform actions on behalf of the store owner.
+
+Environment variables are also available as an alternate configuration method:
```
Comment (madhushreeray30) [inline docs/index.md:82]:
```suggestion
Alternatively, you can also use the standard Shopify environment variables to obtain credentials **only if other arguments (`shop_name ` and `token `) are not specified** in the connection:
```

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+- `SHOPIFY_SHOP_NAME`
+- `SHOPIFY_API_TOKEN`
```
Comment (madhushreeray30) [inline docs/index.md:85]:
```suggestion
```sh
export SHOPIFY_SHOP_NAME= theshop
export SHOPIFY_API_TOKEN=shpat_ab0a4zaa19c3faketoken924176b387d
```

**Review by madhushreeray30 — CHANGES_REQUESTED** [review]
Please have a look at the suggested changes. Thanks!

**Review by madhushreeray30 — CHANGES_REQUESTED** [review]
Please have a look at the suggested changes. Thanks!

File: shopify/shopify_draft_order.go [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+	for _, draftOrder := range draftOrders {
+		d.StreamListItem(ctx, draftOrder)
+
+		if d.RowsRemaining(ctx) == 0 {
+			return nil, nil
+		}
+	}
+	return nil, nil
+}
+
```
Comment (madhushreeray30) [inline shopify/shopify_draft_order.go:188]:
```suggestion

// HYDRATE FUNCTION
```

**Review by misraved — APPROVED** [review]

**Review by misraved — APPROVED** [review]

**Pull request merged**